pyproject.toml

```diff
@@ -22,6 +22,8 @@ openfermion = "^1.6.1"
 ipyparallel = "^8.8.0"
 furo = "^2024.8.6"
 myst-nb = "^1.1.2"
+sphinx-autodoc-typehints = "^2.4.4"
```

Comment:
These dependencies aren't really used here but I've added them as they are also used by the extensions docs. See the `extensions` list variable in `conf.py` maybe I could get around this by altering the extensions list depending on what the parent repository is but that didn't seem very nice.